sparkles/yoshi.py

```diff
@@ -0,0 +1,220 @@
+import numpy as np
+import numpy.ma
+from chandra_aca.transform import calc_aca_from_targ
+from chandra_aca.drift import get_aca_offsets
+import proseco
+from mica.archive.cda import get_ocat_web, get_ocat_local
+from Ska.Sun import nominal_roll
+from cxotime import CxoTime
+
+
+def get_ocat_obsid_date(obsid, web_ocat=True):
```

Comment:
This was a convenience that doesn't seem to be used in the current code so can be cut.